gui/screens/community.py

```diff
@@ -423,10 +423,21 @@ def channel_welcome_note(self) -> str:
 class Members(QObject):
 
     def __init__(self):
-        super().__init__(communities_names.mainWindow_UserListPanel)
+        super().__init__(communities_names.mainWindow_userListPanel_StatusListView)
         self._member_item = QObject(communities_names.userListPanel_StatusMemberListItem)
+        self._user_badge_color = QObject(communities_names.statusBadge_StatusBadge)
 
     @property
     @allure.step('Get all members')
     def members(self) -> typing.List[str]:
         return [str(member.statusListItemTitle.text) for member in driver.findAllObjects(self._member_item.real_name)]
+
+    @allure.step('Verify member is offline by index')
+    def member_is_offline(self, index: int) -> str:
+        self._member_item.real_name['index'] = index
+        return self._user_badge_color.object.color.name == '#939ba1'
+
+    @allure.step('Verify member is online by index')
+    def member_is_online(self, index: int) -> str:
+        self._member_item.real_name['index'] = index
+        return self._user_badge_color.object.color.name == '#4ebc60'
```

Comment:
maybe it makes sense to keep color values somewhere in enum? i think we have one somewhere

Reply:
Yes, you are right! Fixed